.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/code
+    docker:
+      - image: circleci/node:8.11
```

Comment:
Node 8 is already in maintenance stage and won't be maintained in 2020, maybe we should test it with the version 10 or maybe the 12 ?